### DISCUSSION TITLE
Adding custom Livewire component to filament panel

### DISCUSSION THREAD

**Discussion by Umar-Farooq-Shafi** [filamentphp/filament#12194] [General]
I want to add the Livewire Component to the filament panel

![image](https://github.com/filamentphp/filament/assets/62417503/80276262-90e1-4f31-8f87-4a41c4aba206)

getting error

`Livewire page component layout view not found: [components.layouts.app]`

![image](https://github.com/filamentphp/filament/assets/62417503/c9be43fc-82ca-4177-8c68-03a7d0fea606)

inside `web.php`

`Route::get('/admin/suppliers/{record}/line-items', ListSupplierLineItem::class);`

**Comment by leandrocfe** [comment]
If you want to create a page inside the panel, you should use this
https://filamentphp.com/docs/3.x/panels/pages#creating-a-page
